core/iwasm/common/gc/gc_common.c

```diff
@@ -53,6 +81,40 @@ wasm_func_type_get_param_count(WASMFuncType *const func_type)
     return 0;
 }
 
+bool
+wasm_ref_type_normalize(wasm_ref_type_t *ref_type)
+{
+    wasm_value_type_t value_type = ref_type->value_type;
+    int32 heap_type = ref_type->heap_type;
+
+    if (value_type < VALUE_TYPE_NULLREF || value_type > VALUE_TYPE_I32) {
```

Comment:
> Had better
> 
> ```c
> if ((value_type >= VALUE_TYPE_V128 && value_type <= VALUE_TYPE_I32)
>     || (value_type >= VALUE_TYPE_NULLREF && value_type <= VALUE_TYPE_FUNCREF))
> ```
> 
> Since there are no value types between VALUE_TYPE_FUNCREF (0x70) and VALUE_TYPE_V128 (0x7B).

Get it, thanks